src/pages/Detail/Detail.js

```diff
@@ -6,24 +6,23 @@ import { faHeart as solidHeart } from '@fortawesome/free-solid-svg-icons';
 import { faShareSquare as regularShare } from '@fortawesome/free-regular-svg-icons';
 import ImageSlider from './ImageSlider';
 
+import ColorList from './ColorList';
+import SizeList from './SizeList';
 import Nav from '../../components/nav/Nav';
 import Footer from '../../components/footer/Footer';
 import TopDetail from '../../components/top/TopDetail';
 
 function Detail() {
+  const [product, setProduct] = useState([]);
   const [productName, productNameSet] = useState('');
   let productColor = [];
-
   let productColorHex = [];
-
-  let productSizePerColor = {};
-
+  let productSize = [];
   const [productPrice, productPriceSet] = useState(1000);
   let productImgUrl = [];
   const [imgUrl, urlSetting] = useState([]);
-  // const [colorHex, colorHexSet] = useState([]);
 
-  const [idValue, idSet] = useState(5);
+  const [idValue, idSet] = useState(1);
   useEffect(() => {
     fetch('http://localhost:8000/products/details', {
```

Comment:
```suggestion
    fetch(POST_SIGNIN_API, {
```
이런식으로 변경을 하시는게 좋을 것 같습니다.
다른분들 코드 보시면 fetch에 위와 같이 변수로 들어가 있을거예요.
그리고 config.js를 보면 해당 변수가 export 되어 있구요~

Reply:
알겠습니다!